src/components/file-input/__internal__/file-upload-status/file-upload-status.style.tsx

```diff
@@ -65,6 +65,7 @@ export const StyledFileUploadStatus = styled.div<StyledFileUploadStatusProps>`
       ${hasError &&
       `&& ${StyledTypography} {
         color: var(--${colorToken});
+        font-weight: 500;
```

Comment:
Comment: This could be `fontWeights500`

Reply:
We apparently can't do this yet due to an ongoing issue with the font-weight tokens - so I'll leave this as it is for now and we'll update this later along with all the other components using hard-coded font weights.